src/styles/utils/reser.scss

```diff
@@ -0,0 +1,5 @@
+body,
+h1,
+p {
```

Comment:
It is recommended to include more elements in the reset styles to ensure consistent styling across different browsers. Consider adding more selectors like 'h2', 'h3', 'ul', etc.